core/src/main/scala/org/bitcoins/core/protocol/script/ScriptSignature.scala

```diff
@@ -126,14 +126,14 @@ object P2PKHScriptSignature extends ScriptFactory[P2PKHScriptSignature] {
   def isP2PKHScriptSig(asm: Seq[ScriptToken]): Boolean =
     asm match {
       case Seq(_: BytesToPushOntoStack,
-               _: ScriptConstant,
+               maybeSig: ScriptConstant,
                _: BytesToPushOntoStack,
-               z: ScriptConstant) =>
+               maybeKey: ScriptConstant) =>
         if (
-          (z.bytes.length == 33 || z.bytes.length == 65) && ECPublicKey
-            .isFullyValid(z.bytes)
+          (maybeKey.bytes.length == 33 || maybeKey.bytes.length == 65) &&
+          (maybeSig.bytes.length > 68 && maybeSig.bytes.length < 73)
```

Comment:
Just double checking, shoudl these be inclusive or exclusive? i.e. 

```scala
(maybeSig.bytes.length >= 68 && maybeSig.bytes.length <= 73)
```

Reply:
exclusive, 68 is way less than is really probable and 72 is the max sig size

Reply:
72 bytes is the upper bound 

https://github.com/bitcoin/bips/blob/master/bip-0062.mediawiki#der-encoding

I am not sure about the lower bound